ccw.core.test/src/clj/ccw/util_test.clj

```diff
@@ -0,0 +1,59 @@
+(ns ccw.util_test
```

Comment:
i know copyright may be missing from existing files, but can you please add one for this? thanks in advance